code_gen/helpers/langchain_helpers.py

```diff
@@ -0,0 +1,79 @@
+import os
+import openai
+import templates.generate as templates
+from langchain import LLMChain
+from langchain.agents import ZeroShotAgent, AgentExecutor
+from langchain.chat_models import ChatOpenAI
+from langchain.agents.agent_toolkits.json.toolkit import JsonToolkit
+from langchain.tools.json.tool import JsonSpec
+from dotenv import load_dotenv
+load_dotenv()
+
+
+class OpenAPIExplorerTool:
+    @staticmethod
+    def create_tools(docs):
+        json_spec = JsonSpec(dict_=docs, max_value_length=10000)
```

Comment:
Why 10,000?

Reply:
Actually I don't remember. I think it was related to langchain / OpenAI's context window. I can add it back if needed